waffle-provider/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@ethereum-waffle/provider",
   "description": "A mock provider for your blockchain testing needs.",
-  "version": "2.4.0",
+  "version": "2.4.1",
```

Comment:
No changes were made to the provider I think

Reply:
Yeah, it easier for me to just bump all (need to add automatic bumping).